vnext/ReactWindows-UWP.sln

```diff
@@ -153,10 +153,18 @@ Global
 		{11C084A3-A57C-4296-A679-CAC17B603144}.Release|x86.Build.0 = Release|Win32
 		{6F713CA6-6BFE-4DF8-932D-4E49745C967E}.Debug|ARM.ActiveCfg = Debug|Win32
 		{6F713CA6-6BFE-4DF8-932D-4E49745C967E}.Debug|x64.ActiveCfg = Debug|x64
+		{6F713CA6-6BFE-4DF8-932D-4E49745C967E}.Debug|x64.Build.0 = Debug|x64
```

Comment:
Just to confirm, this change re-enables Universal Unit Tests, correct?

Reply:
Yes.